src/db/emoji_logging.rs

```diff
@@ -0,0 +1,73 @@
+use anyhow::*;
+
+use super::Db;
+
+use serenity::model::id::EmojiId;
+
+pub struct EmojiData {
+    //You could store the emoji itself in here
+    //    emoji: Emoji,
```

Comment:
what'd be the advantages/disadvantages of doing so?

Reply:
1, i didnt use it, so the build failed because of clippy
2. parsing emojis from text returns an EmojiIdentifer which isnt actually an emoji and is annoying to do another api request to get the extra details inside the struct

Reply:
ahh, fair. then I'd say remove it completely, including the comment :D